tests/test_math.py

```diff
@@ -75,6 +75,19 @@ def test_construct_float64_random(self):
         self.assert_allclose64(cplx.imag, self.imag1_64)
 
     def test_operator_add_float32(self):
+        cplx = mm.complex64(self.real1_32, self.imag1_32)
```

Comment:
Add unit testing for the `complex @ real` and `real @ complex`.

Reply:
@j8xixo12 , it is inconvenient to repeat the code for both 32-bit and 64-bit testing.  I will create an issue for further refactoring that remove the duplication in the way in https://github.com/solvcon/modmesh/blob/525d70c0cb1c11e979213bacfdfd0d09d9655f37/tests/test_universe.py#L272